.circleci/config.yml

```diff
@@ -62,7 +62,7 @@ commands:
             pnpm --version
       - run:
           name: Install js dependencies
-          command: pnpm install
+          command: pnpm install --ignore-scripts
```

Comment:
This seems to fix the out of memory issues in CI but might be making some integrations tests fail, looking more into it.

Reply:
I tried a few updates to this branch, cleaned up the dependencies a bit and removed the playground folder. That branch seems to pass CI https://github.com/apedroferreira/mui-toolpad/pull/1

Reply:
I see, yeah I guess some of the babel packages I copied over might not be needed...
As for the playground, if we get rid of it do you have any alternative in mind? When I tried to make Toolpad Core work in development in projects outside the monorepo I always had some weird issues.

Reply:
We can try figuring out which part of the playground package.json is the culprit by bissecting the dependencies and  see what happens in CI. Timeconsuming, but probably the only way

Reply:
It was probably Next.js having a different version than the docs, should be good now.